src/js/components/MarathonTaskDetailsList.js

```diff
@@ -58,6 +58,14 @@ class MarathonTaskDetailsList extends React.Component {
     return task.status;
   }
 
+  getTimeField(time) {
+    return (
+      <time dateTime={time} title={time}>
+        {new Date(time).toLocaleString()}
```

Comment:
I love the idea of clicking a timestamp to view alternative formats, since it's easy then to copy/paste.
We should be able to draw a short set of rules for how we'll be consistent with displaying timestamps.